documentation/config_docs.md

```diff
@@ -41,6 +45,9 @@ Refer [here](https://docs.github.com/en/free-pro-team@latest/developers/webhooks
 | `status_rules` | status rules config object | all status notifications are ignored |
 | `project_owners` | project owners config object | no project owners are defined |
 | `ignored_users` | list of users to be ignored on all notifications | no user is ignored |
+| `user_mappings` | list of mappings from git email and/or GitHub handle to Slack email | no mapping defined
+
+Note that in `user_mappings`, git email to Slack email mappings are used for status DMs, while GitHub handle to Slack email mappings are used to get Slack mentions in notifications.
```

Comment:
not explained why need to map by handle when email also available, the latter should be always better?

Reply:
Email is only available for commits and pushes, and it's the one from `git config user.email` if I understand correctly, and that's what I meant by "git email" in the readme. So it's possible to match between emails for CI status DMs. In other types of comments we only have access to the GitHub handle. I can make this clearer in documentation.

Reply:
I think @rr0gi meant to explain the design considerations and decisions on the "GitHub handle to Slack email mappings are used to get Slack mentions in notifications" part, i.e. why not use git email to Slack email too?